code/datums/wounds/burns.dm

```diff
@@ -41,7 +41,7 @@
 		return
 
 	if(victim.reagents)
-		if(victim.reagents.has_reagent(/datum/reagent/medicine/spaceacillin))
+		if(HAS_TRAIT(victim, TRAIT_VIRUS_RESISTANCE))
```

Comment:
This seems to have removed the ability to use spaceacillin to prevent infections from worsening in dead people. Assuming that isn't intentional since it isn't mentioned, I felt I should ask here how it might make sense to fix that. Eg reverting this line, or adding the trait to dead people. I believe this also means having the roach set bonus would make people completely invulnerable to burn infections

Reply:
Whoops, replace the line in spaceacillin with `on_mob_add` and `on_mob_delete` instead of `on_mob_metabolize`